controller/lib/core/sensors.cpp

```diff
@@ -85,10 +84,24 @@ static int get_raw_sensor_zero_reading(AnalogPinId pinId) {
   return runningSum / zeroingAvgSize;
 }
 
-void zero_sensors() {
-  blower_disable();
-  Hal.delay(100); // some arbitrary time to wait for the pressure of the system
-                  // to equalize at all points
+void sensors_init() {
+  // Must wait at least 20 ms from power-on-reset for pressure sensors to
+  // warm-up (TODO: according to whom?).
```

Comment:
According to the datasheet!

Reply:
Awesome!

I read through https://www.nxp.com/docs/en/data-sheet/MPXV5004G.pdf but I couldn't find it; do you know offhand what I should ctrl+f for?

Reply:
https://www.nxp.com/docs/en/data-sheet/MPXV7002.pdf
Table 1, last entry.

Reply:
When I was in Oakland yesterday, Edwin sent me https://docs.google.com/spreadsheets/d/1EOa5USxCaV1uuK5RdZMPWIBl7iGKPtVBh_DIlNusR50/edit#gid=0 and said we're using MPXV5004 and MPXV7002 now.

The corresponding table in the MPXV5004 datasheet doesn't have the same row!  Do you know what to make of that?

Reply:
Well uh...that seems bad? I feel if we were going to switch the differential sensor controller team would absolutely have had to be notified (this changes the code!). What's more interesting is that MPXV5002 returns no search results, even from the manufacturer. I think we need to check with Edwin, I don't think he wrote down the write part number for the new sensor.

As for the 5004 not having that entry, yeah this is what's annoying with parts sometimes. It's from the same manufacturer, and same technology as the 7002, so it'd be perfectly reasonable to say they'd have similar warm-up times. Of course, you'd probably want to double check by actually testing the physical sensor in the lab. It's a good enough and close enough figure for alpha.

Reply:
https://respiraworks.slack.com/archives/C0100MWE4V8/p1587936746167900 is out to Edwin.  I'll update this once I hear back.